include/gsl/gsl_assert

```diff
@@ -20,6 +20,20 @@
 #include <exception>
 #include <stdexcept> // for logic_error
 
+//
+// make suppress attributes parse for some compilers
+// Hopefully temporary until suppresion stardization occurs
+//
+#if defined (_MSC_VER)
+#define GSL_SUPPRESS(x) [[gsl::suppress(x)]]
+#else
+#if defined(__clang__)
+#define GSL_SUPPRESS(x) [[gsl::suppress("x")]]
```

Comment:
Has someone opened a bug report against Clang to get this syntax variation fixed?